test/manipulation.test.js

```diff
@@ -215,18 +203,6 @@ describe('manipulation', function() {
         }).catch(done);
     });
 
-    it('should work when called with no data and callback', function(done) {
-      Person.afterCreate = function(next) {
-        this.should.be.an.instanceOf(Person);
-        should.not.exist(this.name);
-        should.exist(this.id);
-        Person.afterCreate = null;
-        next();
-        setTimeout(done, 30);
-      };
-      Person.create();
```

Comment:
It is tested in a few other files.

https://github.com/strongloop/loopback-datasource-juggler/blob/master/test/basic-querying.test.js
https://github.com/strongloop/loopback-datasource-juggler/blob/master/test/common_test.js